recipes/argh/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "argh" %}
+{% set version = "0.26.2" %}
+{% set hash_type = "sha256" %}
+{% set hash = "e9535b8c84dc9571a48999094fda7f33e63c3f1b74f3e5f3ac0105a58405bb65" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - argparse >= 1.1  # [py26]
+
+test:
+  imports:
+    - argh
+
+about:
+  home: https://github.com/neithere/argh
+  license: GNU Lesser General Public License (LGPL), Version 3
+  summary: The Natural CLI.
```

Comment:
Also have asked upstream to include the license files in packages via PR ( https://github.com/neithere/argh/pull/112 ). Please keep an eye on this via an issue in the feedstock. Once it gets corrected in a version we should add the license via `license_file`.